conjur-oss/README.md

```diff
@@ -331,6 +333,43 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 |`ssl.hostname`|Hostname and Common Name for generated certificate and ingress|`"conjur.myorg.com"`|
 |`postgresLabels`|Extra Kubernetes labels to apply to Conjur PostgreSQL resources|`{}`|
 
+### Deploying Without Persistent Volume Support (e.g. for MiniKube, KataCoda)
+Some Kubernetes platforms (e.g. MiniKube and KataCoda) do not have
+out-of-the-box support for StorageClasses or PersistentVolumes. If you
+are Helm installing a Conjur cluster on such a platform, then it is possible
+to install the cluster without persistent storage of Conjur secrets
+configuration and data by using the following chart setting:
+
+    ```
+    --set persistentVolume.create=false
+    ```
+
+Using this flag means that your Conjur policies and secrets will not be
+stored persistently across pod resets, so this is intended to be used
+for experimentation, exploration, or automated testing, and is **not intended
+to be used in production environments**.
+
+### Deploying Without LoadBalancer Support (e.g. for KinD, MiniKube, KataCoda)
+Some Kubernetes platforms (e.g. Kubernetes-in-Docker [KinD], MiniKube
+and KataCoda) do not have out-of-the-box support for LoadBalancers.
+
+For such platforms, one workaround for this is to install a software
+load balancer add-on, such as MetalLB, and assign a pool of routed IPs
+for the software load balancer to use as external IPs. Configuring such
+a software load balancer is considered out-of-scope here. Please refer to
+the [MetalLB documentation](https://metallb.universe.tf) for details.
+
+An alternative to using a software load balancer would be to install the
+Conjur cluster without LoadBalancer support by using the following
+chart setting:
+
+    ```
+    --set service.external.enabled=false
```

Comment:
Unrelated to this PR; I think we could look into the naming we use for this flag. I don't think it communicates clearly what it does.

Reply:
@BradleyBoutcher good point. This is more of a load balancer disable. Even with this set to `false`, services can still be accessed externally using the NodePort and a node IP address.